src/sentry/integrations/bitbucket/repository.py

```diff
@@ -143,8 +143,8 @@ def _format_commits(self, repo, commit_list):
             } for c in commit_list
         ]
 
-    def compare_commits(self, repo, start_sha, end_sha, actor=None):
-        installation = self.get_installation(repo.integration_id, repo.organization_id)
+    def compare_commits(self, repo, start_sha, end_sha, organization_id):
+        installation = self.get_installation(repo.integration_id, organization_id)
```

Comment:
Are BitBucket repository objects not associated with an Organization like they used to be? Just curious why this needs to change.

Reply:
Oh man... you're right! I wasn't fully aware we had that connection. thanks. :)